source/JsMaterialX/README.md

```diff
@@ -74,21 +78,20 @@ cmake --build .
 
 
 ### Output
-After building the project the `JsMaterialX.wasm` and `JsMaterialX.js` files can be found in `./<build_folder>/source/JsMaterialX/`.
+After building the project the `JsMaterialX.wasm` and `JsMaterialX.js` files can be found in the global install directory of this project.
```

Comment:
Can you be a littke more specific? I dont even use that path, I usually copy the bindings from  `./<build_folder>/source/JsMaterialX/`.

Reply:
I've explicitly made it less specific, since the configuration will no longer be controlled by the JS bindings sub-project as soon as this PR is merged: https://github.com/autodesk-forks/MaterialX/pull/1197/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20a

The intention is to explicitly install all assets to a globally managed path. Mentioning the current path here is condemned to get outdated.